js/main.js

```diff
@@ -3,12 +3,13 @@
  * @return {[string]} [formatted date]
  */
 function formattedDate(){
-  var x = new Date(), year = x.getFullYear(), x.getMonth() + 1, date = x.getDate();
+  var x = new Date();
+  year = x.getFullYear(), month=x.getMonth() + 1, date = x.getDate();
```

Comment:
Please add `var` to the variables declared, otherwise they become global variables.